docs/operations/scalable-query-frontend.md

```diff
@@ -5,15 +5,35 @@ weight: 5
 slug: scaling-query-frontend
 ---
 
-Historically scaling the Cortex query frontend has [posed some challenges](https://cortexmetrics.io/docs/proposals/scalable-query-frontend/).  This document aims to detail how to use some of the added configuration parameters to correctly scale the frontend.  Note that these instructions apply in both the HA single binary scenario or microservices mode.
+Historically scaling the Cortex query frontend has [posed some challenges](https://cortexmetrics.io/docs/proposals/scalable-query-frontend/).
+This document aims to detail how to use some of the added configuration parameters to correctly scale the frontend.
+Note that these instructions apply in both the HA single binary scenario or microservices mode.
+
+## Using Query Scheduler
+
+Query scheduler is new service that moves the in-memory queue from query frontend to a separate component.
```

Comment:
Remember doc is here to stay. Try to avoid time-based references, like "new" or "now".

```suggestion
Query scheduler is a service that moves the in-memory queue from query frontend to a separate component.
```